### PR TITLE
Fix enhanced RCE media in Teacher app

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-14443
affects: teacher
release note: Fixed a bug that could cause some media to not load in rich content

Test plan: See ticket. There should be 3 playable media objects.

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/xgw8zoe) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fxgw8zoe)
teacher: [Link](https://rebrand.ly/so3gygp) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fso3gygp)
parent: [Link](https://rebrand.ly/eq57zwu) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Feq57zwu)

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed a bug that could cause some media to not load in rich content

#### Affected Apps: Teacher

[MBL-14443](https://instructure.atlassian.net/browse/MBL-14443)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.36% | 93.36% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CanvasCore/CanvasCore/WhizzyWig/CanvasWebView.js [hunk 1]
```diff
@@ -29,3 +29,40 @@ var checkQuizzesNext = setInterval(function() {
...
+// and the src is a canvas file
+// reload the webview with an authenticated version of the iframe's src
+if (iframes.length == 1 && /\/courses\/\d+\/files\/\d+\/download/.test(iframes[0].src) && iframes[0].id === "cnvs_content") {
+    window.webkit.messageHandlers.loadFrameSource.postMessage(iframes[0].src)
+} else {
+    // Replace all iframes with a button to launch in SFSafariViewController
+    iframes.forEach(iframe => {
+        const replace = iframe => {
+            const a = document.createElement('a')
+            a.textContent = '{$LTI_LAUNCH_TEXT$}'
```
Comment (vanwagonet) [inline CanvasCore/CanvasCore/WhizzyWig/CanvasWebView.js:46]: does this get replaced with the localized text?

Comment (armstrongnate) [reply CanvasCore/CanvasCore/WhizzyWig/CanvasWebView.js:46]: Good call. It's now getting translated but it's not getting styled. I need to make sure the CSS gets loaded outside of PageTemplateRenderer. I think I might want to wait for your changes [here](https://github.com/instructure/canvas-ios/pull/1361) first though. what do you think?

**Pull request merged**